frameworks/kafka/tests/test_sanity.py

```diff
@@ -273,6 +276,16 @@ def test_pods_cli():
     assert service_cli('pods status {}-0'.format(DEFAULT_POD_TYPE), service_name=FOLDERED_SERVICE_NAME)
     assert service_cli('pods info {}-0'.format(DEFAULT_POD_TYPE), service_name=FOLDERED_SERVICE_NAME, print_output=False) # noisy output
 
+@pytest.mark.sanity
+@pytest.mark.metrics
+def test_metrics():
+    def metrics_exist():
```

Comment:
Could just have a `wait_for_any_metrics(service_name, node_name, timeout=600)` function in `sdk_metrics` (containing the current content of these tests), and then have these tests just call `wait_for_any_metrics`.